src/AWS.Deploy.Orchestration/AWS.Deploy.Orchestration.csproj

```diff
@@ -26,9 +26,9 @@
     <PackageReference Include="AWSSDK.S3" Version="4.0.0.1" />
     <PackageReference Include="AWSSDK.AppRunner" Version="4.0.0.1" />
     <PackageReference Include="AWSSDK.SimpleSystemsManagement" Version="4.0.1.1" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
-    <PackageReference Include="Microsoft.TemplateEngine.IDE" Version="5.0.1" />
-    <PackageReference Include="Microsoft.TemplateEngine.Orchestrator.RunnableProjects" Version="5.0.1" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
```

Comment:
updating things to use task/async is due to these version updates is that right?

Reply:
No, this package has been there forever. There are some functionality in there that we use in the deploy tool. I'm just updating it to .NET8